GithubPractice/ViewController.swift

```diff
@@ -29,4 +35,61 @@ class ViewController: UIViewController {
     @IBAction func didClickClear(_ sender: Any) {
         
     }
+    
+    
+    override func didReceiveMemoryWarning() {
+          super.didReceiveMemoryWarning()
+          // Dispose of any resources that can be recreated.
```

Comment:
you don't need to write comments. The code should be self explanatory by itself

Reply:
The `didReceiveMemoryWarning` method currently does nothing.
All it does is call `super.didReceiveMemoryWarning()`. Feel free to remove it.